docker-assets/initialize_httpd.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#
+# Initial setup for httpd upon container startup.
+#
+
+# Define container environment variables needed by the httpd process
+#
+cat - <<!END! >> /etc/sysconfig/httpd
+
+#
+# Container Environment Variables needed
+#
+MANAGEIQ_SERVICE_NAME = "${MANAGEIQ_SERVICE_NAME}"
```

Comment:
I'm ok on attempting to generalize container-httpd now (not ManageIQ specific), I'll update.